ecmd.go

```diff
@@ -121,7 +121,7 @@ func edittext(w *Window, q int, r []rune) error {
 		f := w.body.file
 		err := f.elog.Insert(q, r)
 		if err != nil {
-			warning(nil, err.Error())
+			warning(nil, err.Error()+"\n")
```

Comment:
Do these newlines belong here, or in the warning function directly?

Reply:
They belong here as the newlines are being added in the call to warning in every other occurrence in the code, prehaps if it was entirely redone they'd be added in the warning function but I was just trying to fix the problem with the fewest changes possible.